### DISCUSSION TITLE
Left upper corner with sysstats ans right upper with connections are almost empty

### DISCUSSION THREAD

**Discussion by byx01** [lesovsky/pgcenter#112] [Q&A] [answered]
Hello! Not far ago installed pg 12.7 and now installed pgcenter_0.9.2_linux_amd64.rpm on same host. This is installed on "Linux Postgres 3.10.0-1127.10.1.el7.x86_64 #1 SMP " . Try to watch with pgcenter top -U postgres postgres , but get no numbers in syssystats(left corner) and no numbers on right corner. ps: i have same situations several times in the past with other installations, but others work fine.
![image](https://user-images.githubusercontent.com/99789314/154213955-dfec193c-f6ed-412d-8d30-f509d508a441.png)

**Comment by lesovsky** [comment, marked as answer]
Hmm, try to run pgcenter with black background and white text.

Yep, i reproduced it with white screen. Well as a workaround you can adjust your terminal settings.

**Reply by byx01** [reply]
Thanks, Bro! with black/white i can see now 
![image](https://user-images.githubusercontent.com/99789314/154620986-92a74a10-88a0-40ef-a1f8-885e07de91b3.png)
